docker/dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM ubuntu:18.04
+
+# Ubuntu setup
+ENV TZ=America/Chicago
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+# Setup environment variables for Docker image
+
+# setup home dir
+ENV HOME /root
+
+# Install dependencies from Debian package manager
+RUN apt-get update -y && \
+    apt-get upgrade -y && \
+    apt-get install -y software-properties-common build-essential && \
+    apt-get install -y wget git vim && \
+    add-apt-repository ppa:jonathonf/gcc-9.1 && \
```

Comment:
same here you could add the repo before hand and then install everything in a single time.

Reply:
Also is gcc-9 required ?
I don't know what is the purpose of this docker container, but if it is for CI-testing maybe staying with the normal gcc version is better as it will be what most of the user have....